isacc_messaging/api/isacc_record_creator.py

```diff
@@ -179,24 +179,31 @@ def get_patient(self, patient_id):
         if result is not None:
             return Patient(result)
 
-    def get_general_practitioner_emails(self, pt: Patient) -> list:
+    def get_practitioners_emails(self, patient_id) -> list:
```

Comment:
@hannahburkhardt did a nice job of adding type annotations, let's maintain them.

make that:
```def get_practitioners_emails(self, patient_id: int) -> list:```